Gemfile.lock

```diff
@@ -434,7 +434,7 @@ GEM
     websocket-driver (0.7.1)
       websocket-extensions (>= 0.1.0)
     websocket-extensions (0.1.4)
-    zeitwerk (2.2.2)
+    zeitwerk (2.3.0)
```

Comment:
### [zeitwerk](https://github.com/fxn/zeitwerk)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 2.2.2 → 2.3.0

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **9** commits. See the full changes on [the compare page](https://github.com/fxn/zeitwerk/compare/v2.2.2...v2.3.0).

These are the individual commits:
- (ad26cb0) [allow eager loading after reloading](https://github.com/fxn/zeitwerk/commit/ad26cb0521cfb88fd8fee687dca7eb10e4ccf1bc)
- (0146d4c) [test Ruby 2.7 on CI](https://github.com/fxn/zeitwerk/commit/0146d4c3be3087a887cdfbbce7c697a0386c27f0)
- (af53354) [Merge pull request #103 from tricknotes/ruby-2.7](https://github.com/fxn/zeitwerk/commit/af53354543b943292eed1e29e6927e3771679905)
- (763e3dc) [Micro optimization: `[0...-3] = ''` is faster than `slice!(-…](https://github.com/fxn/zeitwerk/commit/763e3dc0d1d4cecded327e77f7c8f49d0a383c13)
- (527d548) [Merge pull request #105 from Shopify/micro-optim-slice](https://github.com/fxn/zeitwerk/commit/527d5489c9158ee596077d90491f0846a895868c)
- (2f6b32f) [annotates a type [skip ci]](https://github.com/fxn/zeitwerk/commit/2f6b32fd04fc11f9b853e4be7bece23d5f794847)
- (21e4341) [typo [skip ci]](https://github.com/fxn/zeitwerk/commit/21e434139c322b5bfba4bf9b733a27773987c216)
- (32c1cdb) [adds support for collapsing directories](https://github.com/fxn/zeitwerk/commit/32c1cdbca82c2314a951187f1e4719efb20b7557)
- (806795d) [version 2.3.0](https://github.com/fxn/zeitwerk/commit/806795d302840a7e96612b88ff45f231ea4318b0)


</details>